### DISCUSSION TITLE
Meta Thread For Debian based Linux

### DISCUSSION THREAD

**Discussion by majestrate** [oxen-io/lokinet#2097] [General]
please report any issues you encounter with debian based linux (via deb.oxen.io) in this thread. please include lokinet version number, os release, distro name and kernel version.

we will spin off new issues on the github issue tracker for each of them to track their progress.

**Comment by yudai-patronai** [comment]
Using Debian 13.2
i have installed lokinet cli via the instructions from
`https://docs.oxen.io/oxen-docs/products-built-on-oxen/lokinet/guides/installing-on-linux-cli`
and it seems to be running properly. 
```
lokinet.service - LokiNET: Anonymous Network layer thingydoo, client
     Loaded: loaded (/usr/lib/systemd/system/lokinet.service; enabled; preset: >
     Active: active (running) since Sun 2025-11-16 13:29:32 MSK; 5h 17min ago
 Invocation: f2967b7bdd10409b940fae2dca75e8f2
    Process: 49932 ExecStartPost=/usr/sbin/lokinet-resolvconf add /var/lib/loki>
   Main PID: 49921 (lokinet)
     Status: "v0.9.14 client | known/connected: 1547/3 | paths/endpoints 0/0 [ >
      Tasks: 11 (limit: 9129)
     Memory: 61.8M (peak: 64.3M)
        CPU: 2min 38.136s
     CGroup: /system.slice/lokinet.service
             └─49921 /usr/bin/lokinet /var/lib/lokinet/lokinet.ini

```
But nonetheless neither did my ip change on the whatismyip.com , nor am i able to access any sites banned in my country. 
Am i doing anything wrong ?
